mdgeom/tests/test_info.py

```diff
@@ -41,6 +41,33 @@ def test_extract_ugmx():
         assert udata[key] == pytest.approx(reference[key])
 
 
+def test_extract_ucharmm():
+    reference = {
+        "n_atoms": 3341,
+        "Lx": 0,
+        "Ly": 0,
+        "Lz": 0,
+        "alpha": 0,
+        "beta": 0,
+        "gamma": 0,
+        "n_frames": 98,
+        "totaltime": 96.9999914562418,
+        "dt": 0.9999999119200186,
```

Comment:
You could probably set  to 1.0 as this is the “true” Timestep. I think this is only due to float32 precision looking a bit funny. 

If the tests don’t pass with 1.0 then I wouldn’t change because then you have to start a discussion about the importance of doing floating point comparisons correctly—-which is an important topic but perhaps not something you want to discuss at this moment.